content/blog/commit-lock-file.md

```diff
@@ -0,0 +1,133 @@
++++
+title = "Should you commit lock file?"
+slug = "should-you-commit-lock-file"
+date = 2023-09-27T15:48:02+05:30
+image = "/images/2023/commit-lock-file/version-locking-case.png"
+draft = false
+authors = ["Akshay Vadher"]
+description = ""
+tags = ["Guideline", "Best Practice", "Build", "Tooling", "Lock File", "Package Manager", "npm", "yarn", "pnpm", "bun"]
+categories = ["Guideline", "Best Practice", "Software Craftsmanship"]
+type = ""
++++
+
+# _tl;dr;_
```

Comment:
Nit: no need for semicolon at the end. Should be `TL;DR`

Reply:
![image](https://github.com/incubyte/incubyte.github.io/assets/2619644/4e17e9c9-b248-4ab5-9ed8-30af378ccb6e)
or
![image](https://github.com/incubyte/incubyte.github.io/assets/2619644/e5b43e40-313d-4954-81ec-b01f5e2c96d5)
or
![image](https://github.com/incubyte/incubyte.github.io/assets/2619644/88330f04-d5ca-4d6b-acd9-e3b582d9e6b7)
---
If we consider it a header, the header's first character is always the title case.